R/cal-apply.R

```diff
@@ -5,77 +5,160 @@
 #' the estimate columns names, and levels, from the calibration object.
 #' @param .data An object that can process a calibration object.
 #' @param object The calibration object (`cal_object`).
+#' @param prediction (Optional) Column identifier with the prediction.
```

Comment:
We should name this something different to indicator that it is for the hard class predictions. How about `pred_class`?